xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2358,24 +2358,24 @@ xe.admin.passwordReset.step2.title=Reset your password
 xe.admin.passwordReset.step2.newPassword.label=New password
 xe.admin.passwordReset.step2.newPasswordVerification.label=Re-enter new passwod
 xe.admin.passwordReset.step2.submit=Save
-xe.admin.passwordReset.step2.success=The password has been successfully set.
+xe.admin.passwordReset.step2.success=The password has been successfully reset.
 xe.admin.passwordReset.step2.login=Please login to continue \u00BB
 xe.admin.passwordReset.step2.backToStep1=Back to the password reset page \u00BB
 xe.admin.passwordReset.step2.error.emptyPassword=The password cannot be empty.
 xe.admin.passwordReset.step2.error.verificationMismatch=The two passwords do not match.
 xe.admin.passwordReset.step2.error.wrongParameters=Wrong parameters.
-xe.admin.passwordReset.step2.error.noProgrammingRights=This page requires programming rights to work, which currently isn't the case. Please notify an administrator of this problem and try again later.
+xe.admin.passwordReset.step2.error.noProgrammingRights=This page requires programming rights to work, which currently isn't the case. Please notify an administrator for this problem and try again later.
 xe.admin.passwordReset.step2.versionComment.passwordReset=Password was reset
 xe.admin.passwordReset.step2.versionComment.changeValidationKey=Refreshed password reset token
 
 ### XWiki.Configurable - application configuration
 xe.admin.configurable.title=Custom configurable sections
 xe.admin.configurable.macros.title=Macros for custom configurable sections
 xe.admin.configurable.noPermissionThisApplication=You don't have permission to configure this application.
-xe.admin.configurable.applicationAuthorNoAdmin=This configuration cannot be displayed because it was last edited by [[{0}]] who doesn''t have permission to edit this page.
-xe.admin.configurable.cannotLockNoJavascript=This page cannot be locked for editing because Javascript is turned off, for page editing safety, please enable Javascript.
-xe.admin.configurable.configurationClassNonexistant=No class found by the name {0}, can''t display configuration.
```

Comment:
Those double ' are not mistakes, the are part of the syntax (in the end you have only one).

Reply:
ok